### DISCUSSION TITLE
Multiple publishers

### DISCUSSION THREAD

**Discussion by devoren** [bluenviron/mediamtx#2765] [Questions] [answered]
### Question

Hello! I’m newbie on mediamtx. Can I create twitch like platform , where I push stream with webrtc and use ffmpeg to RTSP to HLS. And other users can publish their own live streams. Can mediamtx handle multiple streams from multiple users, and user can see any live stream?

**Comment by aler9** [comment, marked as answer]
Hello, in MediaMTX you can publish as many streams you want and read them with as many readers you want. Streams are separated by giving each a different name or path. Take a look at the README for instructions.

[a comment by an automated account is not shown]
